src/main/java/emt/init/Registry.java

```diff
@@ -1,32 +1,40 @@
 package emt.init;
 
+import cpw.mods.fml.common.Loader;
 import emt.gthandler.common.loader.EMT_GT_Loader;
 import emt.tile.solar.Solars;
 import emt.util.EMTEssentiasOutputs;
 
 public class Registry {
 
-    private static final EMT_GT_Loader gtloader = new EMT_GT_Loader();
+    public static boolean enableGTCompat;
 
     public static void registerPreInit() {
+        enableGTCompat = Loader.isModLoaded("gregtech");
```

Comment:
does this work with gt6? because I am fairly sure this detection would include gt6.

Reply:
Fixed